src/products/components/ProductOrganization/ProductOrganization.tsx

```diff
@@ -130,7 +134,15 @@ const ProductOrganization: React.FC<ProductOrganizationProps> = props => {
             <Typography className={classes.label} variant="caption">
               <FormattedMessage defaultMessage="Product Type" />
             </Typography>
-            <Typography>{maybe(() => productType.name, "...")}</Typography>
+            <Link
+              onClick={() =>
```

Comment:
Since you pass the `href` to the component what is the point of passing `onClick` as well as it will default to use `RouterLink` with `href`?